interview-questions.md

```diff
@@ -7,60 +7,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: An instance variable is when youre creating an instance inside the varaible. Calling on the instances or instantiating is another term when you are calling upon an action. 
 
-  Researched answer:
+  Researched answer: In object-oriented programming with classes, an instance variable is a variable defined in a class, for which each instantiated object of the class has a separate copy, or instance.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: I remember the blocks challenge while i was driving and not navigating that you needed a d0 in the process. 
```

Comment:
correct and if we have a do we need an end. everything in between those is also the ruby block